<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditor</class>
 <widget class="QWidget" name="MapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MapTreeWidget" name="m_pMapTree">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::MoveAction</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="animated">
    <bool>true</bool>
   </property>
   <property name="allColumnsShowFocus">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>75</number>
   </attribute>
   <attribute name="headerHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="headerMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="headerShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="MapTabWidget" name="m_pMapEditor">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>551</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="m_pMapTreeLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>map tree:</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="m_pMappingModeTab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>231</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="m_pTileMapping">
    <attribute name="title">
     <string>Tiles</string>
    </attribute>
    <widget class="TileTab" name="m_pTileTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="TileView" name="m_pTiles">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Tiles</string>
      </attribute>
     </widget>
     <widget class="TileView" name="m_pAutoTiles">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>AutoTiles</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="m_pLeftCurrentTile">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="m_pRHDrawMode">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Pen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fill</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="m_pLHDrawMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Pen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fill</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="m_pRightCurrentTile">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="m_pObjectMapping">
    <attribute name="title">
     <string>Objects</string>
    </attribute>
    <widget class="QTabWidget" name="m_pObjectTabs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>261</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="WorldObjectPrototypeDragTreeWidget" name="m_pWorldObjects">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>WorldObjects</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="m_pDynamicObjects">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>DynamicObjects</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TileTab</class>
   <extends>QTabWidget</extends>
   <header>tiletab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TileView</class>
   <extends>QWidget</extends>
   <header>tileview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>maptree.h</header>
  </customwidget>
  <customwidget>
   <class>MapTabWidget</class>
   <extends>QTabWidget</extends>
   <header>mapview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WorldObjectPrototypeDragTreeWidget</class>
   <extends>QWidget</extends>
   <header>dragtreewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
